cli/cmd/service.go

```diff
@@ -2193,16 +2203,20 @@ func (c *ServicedCli) cmdServiceTune(ctx *cli.Context) {
 	}
 
 	if modified {
-		if service, err := c.driver.UpdateServiceObj(*service); err != nil {
```

Comment:
UpdateServiceObj was created specifically to avoid needing to parse the service into JSON and then reparse it back into a service.

Reply:
Yah this was just git foo. I have removed the changes